be/src/runtime/exec_env.cpp

```diff
@@ -765,6 +765,9 @@ void ExecEnv::destroy() {
     // _query_pool_mem_tracker.
     SAFE_DELETE(_runtime_filter_cache);
     SAFE_DELETE(_driver_limiter);
+    if (LakeServiceBrpcStubCache::getInstance() != nullptr) {
```

Comment:
may need to do the same thing to BrpcStubCache and BrpcHttpStubCache.

Reply:
> may need to do the same thing to BrpcStubCache and BrpcHttpStubCache.

fix BrpcHttpStubCache and BrpcStubCache is deleted before pipeline_timer in function `ExecEnv::destroy()` . So it is no need to call shutdown for BrpcStubCache.